### PR TITLE
Popup window

### PR DESCRIPTION
**Pull request by Dorcas126**
In this project i add the popup window on my portfolio
I used HTML,CSS,JS.
It is responsive on Mobile and desktop.

### CODE REVIEW

**Review by souad988 — CHANGES_REQUESTED** [review]
Hi @Dorcas126 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
DRY js code
Professional README

## Required Changes ♻️
_Great work congratulations your design look great , one small thing to add to make your popup window look exactly like the design is the devider between list of languages and the two buttons in the button of popup mobile version refer to the images below otherwise everything is good 
image (14)
image (15)


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js [hunk 1]
```diff
@@ -10,3 +10,210 @@ const menuLink = document.querySelectorAll('.ul-link');
...
+    Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum 
+    has been the industrys standard dummy text ever since the 1500s, when an unknown 
+    printer took a galley of type and scrambled it 1960s with the relea`,
+    languages: ['html', 'css', 'javascript'],
+  },
+];
+
+const bodyElement = document.querySelector('body');
+
+// const popbar = document.getElementById('myPopup');
```
Comment (wayungi) [inline script.js:119]: - [OPTIONAL] Kindly delete all unused code left as comments. They make your repo dirty 👍🏾

File: script.js [hunk 2]
```diff
@@ -10,3 +10,210 @@ const menuLink = document.querySelectorAll('.ul-link');
...
+    releorem Ipsum is simply dummy text of the printing and typesetting industry. 
+    Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the 
+    releawn printer took a galley of type and scrambled it 1960s with the releaLorem 
+    Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum 
+    has been the industrys standard dummy text ever since the 1500s, when an unknown 
+    printer took a galley of type and scrambled it 1960s with the relea`,
+    languages: ['html', 'css', 'javascript'],
+  },
+];
+
```
Comment (wayungi) [inline script.js:116]: - [ OPTIONAL] You only have one item in the popup array. This makes all your popup windows for all projects to display the same project detail. This was missed by the previous reviewer, I suggest you implement it before moving forward 👍🏾

**Review by wayungi — APPROVED** [review]
Hi @Dorcas126 , 

image

Great work, your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

**Remember: Every finish line 🏁  is the beginning of a new race 🐎** 

## Highlights
- [ ] Tapping the check project button pops up a popup with details about the project 👍🏾 
- [ ] Tapping the close (X) button makes the popup disappear 👍🏾 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**